cons/cxr.lsp

```diff
@@ -583,9 +583,10 @@
     do
       (let ((level (- (length (symbol-name fn)) 2)))
         (eval `(deftest ,(intern
-                          (concatenate 'string
-                            (symbol-name fn)
-                            "-SET-ALT")
+                          (concatenate
+			   #+:eus string
+			   #-:eus 'string
+                            (symbol-name fn) "-SET-ALT")
```

Comment:
>   1.  Enforce the following rules for the test suite:
  Tests should not be modified.

Perhaps we could incorporate those `#+:eus` (perhaps with some case analysis) in the functions?

Reply:
We can (maybe actually should) use them in auxiliary function. Test functions should be left intact, even if they fail.